packages/sdk/src/evm/constants/thirdweb-features.ts

```diff
@@ -60,6 +61,17 @@ export const FEATURE_METADATA = {
   features: {},
 } as const;
 
+export const FEATURE_APPURI = {
+  name: "AppURI",
+  namespace: "appuri",
```

Comment:
Namespace reads wierdly. Maybe `contract.app` instead? Or if we want the URI part then capitalized is better like `contract.appUri`

Reply:
I think all other namespaces were lowercase, assumed it was convention, if it isn't would prefer camelCase myself

Reply:
Looks like we actually only have one other feature namespace that has 2 words to face this problem, and in that case (lower in this file) we use camelCase - if you take a look at the platform fee namespace we do `platformFee` so `appUri` should be good here and then can change it in the actual getter function in `smart-contract.ts` file as well

Reply:
Remember to change the namespace in the codesnippets you have elsewhere too

Reply:
camelcase is the convention, we also have `contract.claimConditions.set()`

question is`contract.appUri` or `contract.appURI` ? i vote for the second to match the contract.

I also don't mind `contract.app.get()` tbh

Reply:
Will change this to appURI

Reply:
Fixed in latest